### DISCUSSION TITLE
Invite users to the private repo to increase velocity

### DISCUSSION THREAD

**Discussion by RiadhKHEDHIRI** [livewire/flux#386] [Feature Requests]
People will be able to PR bug fixes and enhancements.

**Comment by dudelisius** [comment]
As a (paying) user I feel conflicted about this. Sure, it could speed things up, but it could also slow things down. And it could possibly compromise the quality of the product. Watching some of the live streams it becomes really clear how complicated Flux is and the thought and considerations go into the design and execution of Flux. The vision that Caleb and Hugo have is very clear to them. It feels that at this point it is difficult for anybody other than Caleb and Hugo to keep things clean and consistent and keep the vision in check.

Also the speed in which Flux is updated with both bug fixes and new features is really great at the moment.
